src/rank_llm/evaluation/trec_eval.py

```diff
@@ -163,12 +163,13 @@ def main(args):
     context_size = args.context_size
     prompt_mode = args.prompt_mode
     output_filename = f"trec_eval_aggregated_results_{model}_{prompt_mode}.jsonl"
+    rerank_results_dirname = "rerank_results"
     with open(output_filename, "w") as output:
         for dataset in ["dl19", "dl20", "dl21", "dl22", "news", "covid"]:
             for retrieval_method in RetrievalMethod:
                 if retrieval_method == RetrievalMethod.UNSPECIFIED:
                     continue
-                directory = f"rerank_results/{retrieval_method.name}"
+                directory = f"{rerank_results_dirname}/{retrieval_method.name}"
```

Comment:
Please read this one as an arg with default value "rerank_results"

Reply:
@sahel-sh Updated